common/tcbinfo.c

```diff
@@ -509,6 +502,194 @@ oe_result_t oe_parse_tcb_info_json(
     return result;
 }
 
+OE_INLINE uint32_t read_uint32(const uint8_t* p)
+{
+    return p[0] | (p[1] << 8) | (p[2] << 16) | (p[3] << 24);
+}
+
+OE_INLINE uint64_t read_uint64(const uint8_t* p)
+{
+    uint64_t temp = 0;
+    uint64_t result = 0;
+    for (int i = 7; i >= 0; i--)
+    {
+        temp = p[i];
+        result = (result << 8) | temp;
+    }
+    return result;
+}
+/**
+ * type = qe_identity
+ * Schema:
+ * {
+ *     "version" : integer,
+ *    "issueDate" : string,
```

Comment:
Can you update the schema to accurately reflect what the code is doing?
For example, miscselect is a hex string and not a string.

Reply:
updated.